packages/server/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../tsconfig.json",
   "include": ["src", "test"],
   "compilerOptions": {
-    "lib": ["ES2020"],
+    "lib": ["ES2020", "DOM"],
```

Comment:
Hmmmm. How come is this needed?

Reply:
For the tests and the `window` object in `assertNotBrowser`:
https://github.com/trpc/trpc/blob/b838ef2035435719230d895525a6f82adf93c684/packages/server/src/assertNotBrowser.ts#L5-L6

I can suppress the errors in `assertNotBrowser` and create a separate tsconfig for the tests if you like that better

Reply:
Ah, didn't know window was a global in deno. All good then.

Is there a way to do a "deno build" as part of ci to ensure it all works?

Reply:
Considering it relies on a CDN to work with Deno, probably not - at least not without doing some sort of publish. We *could* do a separate build for Deno but that would definitely complicate things (although I'm open to looking into it if you want to go in that direction).

Reply:
Could we use github as a cdn and have a dynamic deno script that is created and run in CI?

Not asking you to do this, just spitballing if it would work

E.g. create a file creates:

```ts
import trpc from `https://raw.githubusercontent.com/trpc/trpc/{{CURRENT_COMMIT_HASH}}/packages/server/src/index.ts`


// [..] start deno server with a little router
```

Then it could ping itself or something.

Reply:
Example: https://raw.githubusercontent.com/trpc/trpc/919758d/packages/server/src/index.ts

Reply:
There would still be other issues, Deno can work with relative file imports anyway. One thing is that in Deno, all imports must have file extensions (`.ts`).